test/samlp.tests.js

```diff
@@ -454,6 +454,45 @@ describe('samlp (unit tests)', function () {
+    var cert = fs.readFileSync(__dirname + '/test-auth0.cer').toString();
+    var buffer = new Buffer(encodedSamlResponse, 'base64').toString();
+    var xml = buffer.toString();
+    //Create version of response without signature
+
+    var stripped = xml
```

Comment:
Can you explain the complex setup here rather than storing a fixture in this format?

Reply:
You mean the encodedSamlResponse? It's a valid SAMLResponse; I'm just half copying how it was written in the previous formats. 
If you're talking about storing malformed SAML Response and fetching then  I haven't done that yet because there's a lot of variations. I suppose we can have a folder that contains a variety of formats that we'd like to test against. It's currently not our priority though.

Reply:
Fair enough. It is just challenging at a glance to understand the state of the test.